controllers/upgrade.go

```diff
@@ -296,25 +354,25 @@ func (r *RollingUpgradeReconciler) IsInstanceDrifted(rollingUpgrade *v1alpha1.Ro
 			upgradeCreationTime = rollingUpgrade.CreationTimestamp.Time
 		)
 		if nodeCreationTime.Before(upgradeCreationTime) {
-			r.Info("rolling upgrade configured for forced refresh", "name", rollingUpgrade.NamespacedName(), "instance", instanceID)
+			r.Info("rolling upgrade configured for forced refresh", "instance", instanceID, "name", rollingUpgrade.NamespacedName())
```

Comment:
Why do we switch the order for instanceID and namespace ?
I think it's better to have instance Name here instread of InstanceID.

Reply:
I did that for the sake of uniformity. In instance manager, old controller, this was the order followed. Namespace name comes in the end.